src/api.js

```diff
@@ -9,14 +9,16 @@ export const geoApiOptions = {
 export const GEO_API_URL = "https://wft-geo-db.p.rapidapi.com/v1/geo";
 export const WEATHER_API_URL = "https://api.openweathermap.org/data/2.5"
 export const WEATHER_API_KEY = "7454d65faa975f224f6bc598ccfe852f"
-// export const DISASTER_API_URL = "https://dc6e72c4-8622-4280-9089-79102851df02.mock.pstmn.io/api/v1/disasters?lat=33.2896&long=-97.6982" //MOCK DATA
 
-export const DISASTER_API_URL = "https://stay-or-go-server.herokuapp.com/api/v1"
-// export const DISASTER_API_URL = "https://stay-or-go-server.herokuapp.com/api/v1/disasters?lat=34.382692&long=-94.805750"
+export const DISASTER_API_URL = "https://stay-or-go-server.herokuapp.com/api/v1/disasters"
+export const DISASTER_MOCK_URL = "https://dc6e72c4-8622-4280-9089-79102851df02.mock.pstmn.io/api/v1/disasters?lat=33.2896&long=-97.6982"
```

Comment:
⛏️ This looks good! Are we sure we want to hard-code the `long` and `lat`?

Reply:
Now that I see how `DISASTER_API_URL` is used below, I think the only suggestion I might have is to move your mock URLs into a different file.<img src="https://media4.giphy.com/media/uWYjSbkIE2XIMIc7gh/giphy-downsized-medium.gif"/>